selfdrive/locationd/locationd.py

```diff
@@ -183,10 +183,9 @@ def locationd_thread(sm, pm, disabled_logs=[]):
 
     if localizer.filter_ready and sm.updated['gpsLocationExternal']:
       t = sm.logMonoTime['gpsLocationExternal']
-      msg = messaging.new_message()
+      msg = messaging.new_message('liveLocation')
       msg.logMonoTime = t
 
-      msg.init('liveLocation')
```

Comment:
This is causing the merge conflict. Does it matter that you initialize the message after setting the `logMonoTime`?

Reply:
No that doesn't matter.